python_transport/wirepas_gateway/utils/argument_tools.py

```diff
@@ -290,6 +290,56 @@ def add_mqtt(self):
             help=("When true the client will skip the TLS check."),
         )
 
+        self.mqtt.add_argument(
+            "--mqtt_reconnect_delay",
+            default=0,
```

Comment:
Could we add env handling in here as well? Unless we merge https://github.com/wirepas/gateway/pull/137 afterwards

Reply:
Yes, it depends in which order we merge them.
What about the other "buffering" settings? I guess we should add one also for them

Reply:
Yes, it applies to the others as well. 

Lets tackle it on #137, makes more sense since it is not yet merged.